code/Compass.php

```diff
@@ -310,9 +321,20 @@ public function rebuild($verbose = false)
                     continue;
                 }
                 
+                // Skip other modules
+                if ($name == 'framework') {
```

Comment:
Why skip these core modules? If you're contributing to those modules you'd want them recompiled.